### PR TITLE
Feat/916 cypress test on seedlot detail page

### LINKED ISSUE #916: cypress test on seedlot detail page
Implement a Cypress test on seedlot detail page `/seedlots/details/{seedlot_number}`

Useful commands:
`npm run cy:open`

- [x] All relative information should be rendered
- [x] Can change applicant seedlot info
- [x] All tests pass

### PR DESCRIPTION
**Pull request by ngunner15**
# Description
Closes #916 

#### New
-frontend/cypress/e2e/smoke-test/edit-applicant-seedlot-info.cy.ts,
-frontend/cypress/e2e/smoke-test/seedlot-detail.cy.ts,
-frontend/cypress/fixtures/default-seedlot-detail.json,
-frontend/cypress/fixtures/user_seedlots.json,
-frontend/cypress/fixtures/applicant-info-change.json,
-frontend/cypress/fixtures/forest-clients.json,
frontend/cypress/fixtures/seedlot-source.json

#### Changed
-frontend/cypress.config.ts,
-frontend/cypress/e2e/smoke-test/seedlot-main-page.cy.ts,
-frontend/cypress/support/e2e.ts,

#### Removed
-

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [x] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-19-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-19-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-19-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/cypress/e2e/smoke-test/seedlot-detail.cy.ts
Comment (ngunner15) [inline frontend/cypress/e2e/smoke-test/seedlot-detail.cy.ts:1]: I tested the title and button functionality.
If we want to add more tests I can add tests on all the fields of this page.

**Review by DerekRoberts — CHANGES_REQUESTED** [review]
Wow, we need this so badly!

Please don't forget to include the tests in our workflows.  Want a hand?

**Comment by ngunner15** [comment]
@DerekRoberts How can I include the tests in the workflow?

**Review by DerekRoberts — APPROVED** [review]
Sorry @ngunner15!  Tests are already loaded.  Now approved.  :)

**Pull request merged**